## Genome assembly
In this study we generated four haploid genome assemblies derived from two interspeicifc hybrids. The scripts can be found in [Assembly]. The details of the methods are avaiable in the supplementary methods.


## Genome annotation
We used RNA-seq (short read) and Iso-Seq (long read) to annotate the transcripts. Together with the protein sequences of the published amphioxus genomes, we annotated the coding genes in the amphixous genomes produced in this study. The scripts are availabe in [Annotation].


## Sex chromosome identification
We used individually re-sequenced data of male and female population to identify the sex-linked regions. The sex-linked regions show the higherst levels of population differentiation between males and females. The scripts used for those analyses are available in [Sex chromosome]. 

## Other analyses
This study also includes analyses of [segmental duplications], [TADs], [Hox gene evolution].

[Assembly]:https://github.com/lurebgi/amphioxusGenome/tree/master/Assembly
[Annotation]:https://github.com/lurebgi/amphioxusGenome/tree/master/annotation
[Sex chromosome]:https://github.com/lurebgi/amphioxusGenome/tree/master/Sex-linkage
[segmental duplications]:https://github.com/lurebgi/amphioxusGenome/tree/master/segmental_duplication
[TADs]:https://github.com/lurebgi/amphioxusGenome/tree/master/Hi-C
[Hox gene evolution]:https://github.com/lurebgi/amphioxusGenome/tree/master/Hi-C/Hox
